doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -496,6 +496,18 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-9-0
+KRKNWK-19846: The wrong command for the internal configuration build in the :ref:`matter_template_sample` sample documentation.
+  There is an obsolete and wrong command for building the sample for nRF54L15 DK with support for Matter OTA DFU and DFU over Bluetooth SMP, and using internal MRAM only.
+
+  **Affected platforms:** nRF54L15
+
+  **Workaround:** Use the following command to build the sample for nRF54L15 DK with support for Matter OTA DFU and DFU over Bluetooth SMP, and using internal MRAM only:
+
+  .. code-block:: console
+
+    west build -p -b nrf54l15dk/nrf54l15/cpuapp -- -DCONFIG_CHIP_DFU_OVER_BT_SMP=y -DFILE_SUFFIX=internal
```

Comment:
```suggestion
.. rst-class:: v2-9-0

KRKNWK-19846: The wrong command for the internal configuration build in the :ref:`matter_template_sample` sample documentation
  There is an obsolete and wrong command for building the sample for the nRF54L15 DK with support for Matter OTA DFU and DFU over Bluetooth SMP, and using internal MRAM only.

  **Affected platforms:** nRF54L15

  **Workaround:** Use the following command to build the sample for the nRF54L15 DK with support for Matter OTA DFU and DFU over Bluetooth SMP, and using internal MRAM only:

  .. code-block:: console

     west build -p -b nrf54l15dk/nrf54l15/cpuapp -- -DCONFIG_CHIP_DFU_OVER_BT_SMP=y -DFILE_SUFFIX=internal
```

Reply:
Are you sure that 

`**Affected platforms:** nRF54L15` has the same indention as `KRKNWK-19846: Wrong command for the internal configuration build in the` and not as `There is an obsolete and wrong command for building the sample for the nRF54L15 DK with support for Matter OTA DFU and DFU over Bluetooth SMP, and using internal MRAM only.` ?

I can see that only the entry for `Enhanced ShockBurst (ESB)` has a different indention (like you suggested).

Reply:
You are right about that. I messed up with the indentations of Affected platforms, Workaround and the code block that follows. Please have two spaces before all of those.